contrib/slack.php

```diff
@@ -76,8 +76,8 @@
 
 // Deploy message
 set('slack_text', '_{{user}}_ deploying `{{target}}` to *{{hostname}}*');
-set('slack_success_text', 'Deploy to *{{target}}* successful');
-set('slack_failure_text', 'Deploy to *{{target}}* failed');
+set('slack_success_text', 'Deploy of `{{target}}` to *{{hostname}}* successful');
```

Comment:
Showing hostname is not the best practise. We ascully want to show something like Deploy prod. Or deploy staging.

Reply:
> Deploying to two host in one go?

Yes. 

OR just a host name with IP. And we better to use aliaas in this case.

Reply:
I guess we need a special magic config to determine the value of deployed hosts.